src/Android/Xamarin.Android/Samples/MapView/TakeScreenshot/TakeScreenshot.cs

```diff
@@ -54,29 +55,28 @@ private void Initialize()
             try
             {
                 // Export the image from map view
-                var exportedImage = await _myMapView.ExportImageAsync();
```

Comment:
It looks like the extension method was used properly here, so I don't think it makes sense to change. I recommend reverting the changes to this sample for Android.